faq.md

```diff
@@ -155,7 +162,7 @@ fn is_odd(x: i64) -> bool {
 }
 ```
 
-In each example, the last line of the function is the return value of that function. It is important to note that if a function ends in a semicolon, its return type will be `()`, indicating no returned value. Implicit returns must omit the semicolon to work.
+In each example, the last line of the function is the return value of that function. It is important to note that if a block ends in a semicolon, its return type will be `()`, indicating no returned value. Implicit returns must omit the semicolon to work.
```

Comment:
Here you changed 'function' to 'block' while the rest of the sentence talks about 'return' types. I don't typically think of blocks as 'returning' something and haven't seen 'return' used this way. The preceding sentence is also talking about functions.